utils/installer.sh

```diff
@@ -60,8 +60,10 @@ colors_ () {
 #   Defaults to 'https://github.com/bbatsov/prelude.git'
 # -i/--into
 #   If one exists, install into the existing config
+# -m/--no-move-dotemacs
+#   Leave '$HOME/.emacs' untouched
 # -n/--no-bytecompile
-#   Skip the compilation of the prelude files.
```

Comment:
This was just done for consistency with the rest of the one-line-description args documented in the comment block.